IPython/core/interactiveshell.py

```diff
@@ -220,6 +317,23 @@ class InteractiveShell(SingletonConfigurable):
         """
     ).tag(config=True)
 
+    autoawait = Bool(True, help=
+        """
+        Automatically run await statement in the top level repl.
+        """
+    ).tag(config=True)
+
+    loop_runner_map ={
+        'asyncio':_asyncio_runner,
+        'curio':_curio_runner,
+        'trio':_trio_runner,
```

Comment:
It seems a little odd that tornado, the only async runner that we actually use in IPython, isn't supported here. Is there a reason for that?